arrangor/src/main/kotlin/no/nav/amt/tiltak/veileder/ArrangorVeilederDbo.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.amt.tiltak.veileder
+
+import no.nav.amt.tiltak.core.domain.arrangor.ArrangorVeileder
+import java.time.ZonedDateTime
+import java.util.*
+
+data class ArrangorVeilederDbo(
+	val id: UUID,
+	val ansattId: UUID,
+	val deltakerId: UUID,
+	val erMedveileder: Boolean,
+	val gyldigFra: ZonedDateTime,
+	val gyldigTil: ZonedDateTime,
```

Comment:
Har vi alltid en gyldigTil-dato for veilederen?

Reply:
Ja, den settes som 3000-01-01 når den opprettes, og settes til now() når den skal inaktiveres.